site/src/pages/release-notes/1.14.0.mdx

```diff
@@ -0,0 +1,348 @@
+---
+date: 2022-1-27
+---
+
+## 🌟 New features
+
+- You can now use [Scala 3](https://dotty.epfl.ch/) for `armeria-scala` and `armeria-scalapb`. #3614 #4036
+  <Tip>
+
+  Scala 3 modules are currently experimental. We're looking forward to your feedback.
+
+  </Tip>
+
+- You can now fluently convert an <type://HttpResponse> into a desired type using <type://WebClient>. #4021
+  ```java
+  WebClient client = WebClient.of("https://api.example.com");
+  CompletableFuture<ResponseEntity<MyObject>> response =
+      client.prepare()
+            .get("/v1/items/1")
+            .asJson(MyObject.class)
+            .execute();
```

Comment:
We use 4-space indentation in some places but 2-space one in other places. Given people can read this document on a mobile device, how about using 2-space indentation with aggresive line breaks?

Reply:
I intentionally use 4 spaces because it is a continuous indentation that has 8 spaces in an IDE based on 4 spaces.

Reply:
On second thought it would be better to use 2-space indentation always regardless of the previous context for mobile devices.